charts/castai-evictor/README.md

```diff
@@ -12,9 +12,13 @@ Cluster utilization defragmentation tool
 | affinity.nodeAffinity.requiredDuringSchedulingIgnoredDuringExecution.nodeSelectorTerms[0].matchExpressions[0].values[0] | string | `"windows"` |  |
 | aggressiveMode | bool | `false` | Specifies whether the Evictor can behave as aggressive if true, evictor will start considering single replica pods as long as they can be scheduled somewhere else. |
 | apiKeySecretRef | string | `""` | Name of secret with Token to be used for authorizing agent access to the API apiKey and apiKeySecretRef are mutually exclusive The referenced secret must provide the token in .data["API_KEY"]. |
+| clusterVPA | object | `{"enabled":true,"pollPeriodSeconds":300,"repository":"registry.k8s.io/cpa/cpvpa","resources":{},"version":"v0.8.4"}` | Cluster proportional vertical autoscaler for the agent deployment https://github.com/kubernetes-sigs/cluster-proportional-vertical-autoscaler. |
```

Comment:
Might want to change agent to evictor in the description (`Cluster proportional vertical autoscaler for the evictor deployment`). Could also update other properties where it mentions agent instead of evictor